fs_storage/models/fs_storage.py

```diff
@@ -242,6 +242,8 @@ def _get_protocols(self) -> list[tuple[str, str]]:
                 protocol.append((p, f"{p} ({cls.__name__})"))
             except ImportError as e:
```

Comment:
Could this syntax be more efficient?
```suggestion
            except (ImportError, KeyError) as e:
                   _logger.debug("Cannot load the protocol %s. Reason: %s", p, e)
```